Wikipedia/Code/ArticleWebMessagingController.swift

```diff
@@ -637,13 +637,6 @@ extension ArticleWebMessagingController {
                         seContainer.style.paddingRight = "\(layoutMargins.right)px";
                  }
 
-                 if (skeletonContainer) {
```

Comment:
Can we also remove the `var skeletonContainer = document.getElementById('\(articleAsLivingDocBoxSkeletonContainerID)');` line above?

Reply:
👍  yep, this has been removed now.